<launch>

  <arg name="debug" default="false" />
  <arg name="path" default="~/objects"/> 

  <arg unless="$(arg debug)" name="launch_prefix" value="" />
  <arg     if="$(arg debug)" name="launch_prefix" value="gdb -ex run --args" />

  <node name="learning" pkg="simple_recognition" type="learning" output="screen" launch-prefix="$(arg launch_prefix)">
    <param name="lua_files" value="$(find simple_recognition)/Lua/"/>
  	<param name="setup_files" value="$(arg path)/"/>
  </node>


</launch>
